packages/cf-deploy-config-inquirer/src/prompts/conditions.ts

```diff
@@ -1,30 +1,26 @@
-import type { CfDeployConfigAnswers } from '../types';
-
-/**
- * Determines if the destination question should be shown based on the target configuration.
- *
- * @param {Answers} previousAnswers - The answers provided in previous prompts.
- * @returns {boolean} `true` if the target is Cloud Foundry or not yet defined, otherwise `false`.
- */
-export function showDestinationQuestion(previousAnswers: CfDeployConfigAnswers): boolean {
-    return previousAnswers.targetName === undefined || previousAnswers.targetName === 'cf';
-}
-
 /**
  * Determines whether to show the managed application router question based on the following conditions:
  * - The target is Cloud Foundry.
  * - No mta yaml configuration is found at the given path.
  * - The project is not a CAP project.
  *
  * @param mtaYamlExists
- * @param {Answers} previousAnswers - The answers provided in previous prompts.
  * @param {boolean} isCapProject - Indicates if the project is a CAP project.
- * @returns {Promise<boolean>} A promise that resolves to `true` if the question should be shown, otherwise `false`.
+ * @returns {boolean} Resolves to `true` if the question should be shown, otherwise `false`.
+ */
+export function showManagedAppRouterQuestion(mtaYamlExists: boolean, isCapProject: boolean): boolean {
```

Comment:
These functions should be inline, there's no need to add functions for simple when clauses it makes the code harder to read.

Reply:
Now displaying `managedAppRouter` only if [enabled](https://github.com/SAP/open-ux-tools/pull/2436/files#diff-434b3a94822e0e4c82bb418db3b7b73e1268fbaaa7b47c2fbd0401df5cc366bdR131)